packages/core/admin/ee/server/services/__tests__/audit-logs-service.test.js

```diff
@@ -96,8 +96,8 @@ describe('Audit logs auth', () => {
       auditLogsService.register();
 
       expect(() => {
```

Comment:
Tiny thing but I just saw on the docs that when using .rejects you have to await the expect